src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -4077,10 +4077,6 @@ class C
                 state.SendTypeChars("w")
                 Await state.AssertSelectedCompletionItem(displayText:="when", isHardSelected:=False)
 
-                state.SendBackspace()
-                state.SendTypeChars("i")
-                Await state.AssertSelectedCompletionItem(displayText:="identifier", isHardSelected:=False)
```

Comment:
❓  Why did this change?

Reply:
Good catch - This change was from the previous revision which removed any suggestions from fields and properties as well.  Missed adding it back when we changed to locals as well.  Adding it back